backend/src/services/instance/instance.entity.ts

```diff
@@ -13,6 +14,7 @@ class Instance {
   public awsInstanceId: string;
   public tenantId: string;
   public state: InstanceState;
+  public operationSystem: SCInstanceRelatedItem;
```

Comment:
Dto is used **only** for sharing API responses types with frontend. You should never use dto for any other types. If you need backend specific enums/types you must create them on backend only. You can either use src/common/enums folder or src/services/instance